data/src/test/java/camp/nextstep/edu/github/data/GithubRepositoryImplTest.kt

```diff
@@ -0,0 +1,52 @@
+package camp.nextstep.edu.github.data
+
+import camp.nextstep.edu.github.domain.GithubRepository
+import com.google.common.truth.Truth.assertThat
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import io.mockk.*
+import org.junit.Before
+import org.junit.Test
+
+@ExperimentalCoroutinesApi
+internal class GithubRepositoryImplTest {
+
+    lateinit var githubRepository: GithubRepository
+    lateinit var githubApiService: GithubApiService
+
+    @Before
+    fun setUp() {
+        githubApiService = mockk()
+        githubRepository = GithubRepositoryImpl(githubApiService)
+    }
+
+    @Test
+    fun 저장소를_불러오는_메서드를_호출했을_때_GithubApiService의_호출되는_메서드를_확인한다() = runTest {
+        //given
+        coEvery { githubApiService.repositories() } returns emptyList()
+
+        //when
+        githubRepository.getRepo()
+
+        //then
+        coVerify { githubApiService.repositories() }
+    }
+
+    @Test
+    fun 저장소를_불러오는_메소드를_호출했을_때_반환되는_값을_확인한다() = runTest {
+        //given
+        val response = listOf<GithubRepoResponse>(GithubRepoResponse(-1, "test", "test description"))
+        coEvery { githubApiService.repositories() } returns response
+        val expected = response.map(GithubRepoResponse::toDomain)
+
+        //when
+        val actual = githubRepository.getRepo()
+
+
+        //then
+        coVerify { githubApiService.repositories() }
+        assertThat(actual.list).isEqualTo(expected)
```

Comment:
```suggestion
        assertThat(actual.list).isEqualTo(expected)
```
이미 위의 테스트 코드에서 verify를 하고 있기 때문에, 여기에서는 verify가 의미가 없어요.